keywords/keywords.robot

```diff
@@ -1,6 +1,6 @@
 *** Settings ***
 Documentation       Template keyword resource.
-
+Variables    variables.py
```

Comment:
shouldn't this be indented with the line above it?

Reply:
that is how LSP formats it

Reply:
sorry it isnt, for some reason formatting was not saved...